pkg/db/db.go

```diff
@@ -112,7 +112,7 @@ func (o Organization) HasKey(key jwk.Key, validAtMoment time.Time) (bool, error)
 			// Found the key
 			chainInterf, chainExists := k.Get("x5c")
 			if chainExists {
-				certificate := chainInterf.([]*x509.Certificate)[0]
+				certificate := chainInterf.(jwk.CertificateChain).Get()[0]
```

Comment:
a transport format that penetrated the DB layer, interesting. future refactor...